docs/pages/eas-update/continuous-deployment.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Continuous Deployment
+description: Learn how to use the fingerprint runtime version for continuous deployment in production and during development.
+---
+
+> **info** This is available in beta for SDK 51 and not yet considered stable. Use may result in unexpected system behavior.
+
+## Concepts
+
+### Fingerprint Runtime Versioning
+
+The [`fingerprint` runtime version policy](/eas-update/runtime-versions/#fingerprint-runtime-version-policy) uses the `@expo/fingerprint` package to generate a hash of your project during build or update operations, and then uses that hash as the runtime version. The hash is calculated based on things like dependencies, custom native code, native projects, and configuration among others.
+
+By automatically calculating the runtime version, one no longer needs to be concerned about about native layer compatibility with the JavaScript application when deploying updates to builds.
+
+## Continuous Integration (CI) and Continuous Deployment (CD)
+
+CI/CD for a React Native project dictates that every JS change (update) is deployed to all compatible builds over-the-air, and new builds are created when the runtime changes.
+
+It can be accomplished by performing roughly these steps:
+1. Run application tests, abort if tests fail.
+2. Calculate fingerprint for relevant platforms using `npx expo-updates fingerprint:generate` command.
+3. Check for EAS builds with a runtime version equal to the fingerprint (`eas build:list --runtimeVersion <fingerprint>`). Kick off a new build if none exists yet.
+    - (optional) Submit to app stores for truly continuous deployment. This has not yet been tested and is not recommended.
+4. Run `eas update` to publish an update for each commit.
+
+### GitHub Actions
+
+Expo provides a [GitHub Action](https://github.com/expo/expo-github-action/tree/main/continuous-deploy-fingerprint) for performing the steps above within a GitHub Actions workflow.
+
+## Debugging fingerprint inconsistencies
+
+If you notice different fingerprints being generated across different machines or environments, it may mean that there are unanticipated files being included in the hash calculation. `@expo/fingerprint` has a predetermines set of files to include/exclude for hash calculation, but often your project setup may require additional excludes. For projects that contain native code directories (`./android` and/or `./ios`) this is more common.
```

Comment:
```suggestion
If you notice different fingerprints being generated across different machines or environments, it may mean that unanticipated files are being included in the hash calculation. `@expo/fingerprint` has a predetermines set of files to include/exclude for hash calculation, but often your project setup may require additional excludes. For projects that include native directories (**android** and **ios**) this is more common.
```

Bold file and directory as per our [writing style guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#filenames-as-bold-text)